client/src/actions/images.js

```diff
@@ -0,0 +1,26 @@
+const ENDPOINT = '/api'
+
+const IMAGE_ENDPOINT = ENDPOINT + '/images'
+
+export const getImages = async (selected) => {
+    const {sharks, cats} = selected
+
+    let url = IMAGE_ENDPOINT
+
+    if (sharks && !cats) {
+        url += '?content=sharks'
+    } else if (!sharks && cats) {
+        url += '?content=cats'
+    }
+    
+    return new Promise((resolve, reject) => {
+        fetch(url)
+            .then(res => {
+                resolve(res.json().then(json => json.images))
+            })
+            .catch(error => {
+                alert(error)
+            })
+    })
```

Comment:
Is this neccessary? Why / why not?

Reply:
Good point, it isn't necessary since `fetch` returns a promise anyway. If I had to refactor this, I would change it to:
```js
return fetch(url)
        .then(res => res.json())
        .then(json => json.images)
        .catch(error => {
            alert(error)
        })
```